packages/image_picker/image_picker/lib/image_picker.dart

```diff
@@ -277,22 +269,77 @@ class ImagePicker {
     int? imageQuality,
     bool requestFullMetadata = true,
   }) {
-    if (imageQuality != null && (imageQuality < 0 || imageQuality > 100)) {
-      throw ArgumentError.value(
-          imageQuality, 'imageQuality', 'must be between 0 and 100');
-    }
-    if (maxWidth != null && maxWidth < 0) {
-      throw ArgumentError.value(maxWidth, 'maxWidth', 'cannot be negative');
-    }
-    if (maxHeight != null && maxHeight < 0) {
-      throw ArgumentError.value(maxHeight, 'maxHeight', 'cannot be negative');
-    }
+    final ImageOptions imageOptions = ImageOptions.createAndValidate(
+      maxWidth: maxWidth,
+      maxHeight: maxHeight,
+      imageQuality: imageQuality,
+      requestFullMetadata: requestFullMetadata,
+    );
 
     return platform.getMultiImageWithOptions(
       options: MultiImagePickerOptions(
-        imageOptions: ImageOptions(
+        imageOptions: imageOptions,
+      ),
+    );
+  }
+
+  /// Returns an [XFile] of the image or video that was picked.
+  /// The image or videos can only come from the gallery.
+  ///
+  /// Where iOS supports HEIC images, Android 8 and below doesn't. Android 9 and
+  /// above only support HEIC images if used in addition to a size modification,
+  /// of which the usage is explained below.
+  ///
+  /// In Android, the MainActivity can be destroyed for various reasons.
+  /// If that happens, the result will be lost in this call. You can then
+  /// call [getLostData] when your app relaunches to retrieve the lost data.
+  ///
+  /// If no images or videos were picked, the return value is null.
+  Future<XFile?> pickMedia({
+    double? maxWidth,
+    double? maxHeight,
+    int? imageQuality,
+    bool requestFullMetadata = true,
+  }) async {
+    final List<XFile> listMedia = await platform.getMedia(
+      options: MediaOptions(
+        imageOptions: ImageOptions.createAndValidate(
+          maxHeight: maxHeight,
           maxWidth: maxWidth,
+          imageQuality: imageQuality,
+          requestFullMetadata: requestFullMetadata,
+        ),
+        allowMultiple: false,
+      ),
+    );
+
+    return listMedia.isNotEmpty ? listMedia.first : null;
+  }
+
+  /// Returns a [List<XFile>] with the images and/or videos that were picked.
+  /// The images and videos come from the gallery.
+  ///
+  /// Where iOS supports HEIC images, Android 8 and below doesn't. Android 9 and
+  /// above only support HEIC images if used in addition to a size modification,
+  /// of which the usage is explained below.
+  ///
+  /// In Android, the MainActivity can be destroyed for various reasons.
+  /// If that happens, the result will be lost in this call. You can then
+  /// call [getLostData] when your app relaunches to retrieve the lost data.
+  ///
+  /// If no images or videos were picked, the return value is an empty list.
+  Future<List<XFile>> pickMultipleMedia({
+    double? maxWidth,
+    double? maxHeight,
+    int? imageQuality,
```

Comment:
Here and in the method above, we should document that these three options are only applied to images.